app/src/main/java/org/dhis2/uicomponents/map/layer/MapLayer.kt

```diff
@@ -16,4 +17,8 @@ interface MapLayer {
     fun findFeatureWithUid(featureUidProperty: String): Feature?
 
     var visible: Boolean
+
+    fun getId(): String {
+        return ""
```

Comment:
???

Reply:
It is the default implementation for the layer. I just wanted it on the new field layer without having to implement it on all other layers, but i guess that would be better